docs/source/_templates/layout.html

```diff
@@ -1,26 +0,0 @@
-{% extends "!layout.html" %}
-
-{# Custom CSS overrides #}
-{% set bootswatch_css_custom = ['_static/custom.css'] %}
-
-{% block relbar1 %}
-<div style="background-color: white; text-align: left; padding: 10px 10px 15px 15px; width=100%; height=auto">
-<a href="{{ pathto('index') }}"><img src="{{
```

Comment:
exact copy of the main template file, except the `path_to` here is changed because of `index.html` instead of index.rst.